.github/workflows/linux-32bit-build-and-test.yaml

```diff
@@ -50,40 +56,75 @@ jobs:
 
     steps:
 
+    # /__e/node16/bin/node (used by actions/checkout@v3) needs 64-bit libraries
```

Comment:
What is `/__e/node16/bin/node`, some nodejs binary?

Reply:
It is the automatically injected node.js binary by GitHub. It is used to execute node based actions like `actions/checkout`..